app/script/util/util.js

```diff
@@ -232,28 +232,40 @@ z.util.base64ToBlob = base64 => {
 /**
  * Downloads blob using a hidden link element.
  * @param {Blob} blob - Blob to store
- * @param {string} filename - Data will be saved under this name
```

Comment:
Filename is a single word these days. No need to camelCase. https://en.wikipedia.org/wiki/Filename